io.github.realmazharhussain.GdmSettings.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: io.github.realmazharhussain.GdmSettings
+command: gdm-settings
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: "42"
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host-os
+  - --filesystem=host-etc
+cleanup:
+  - /bin/blueprint-compiler
+  - /lib/*/site-packages/blueprintcompiler
+  - /include
+  - /lib/pkgconfig
+  - /man
+  - /share/doc
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - '*.la'
+  - '*.a'
+modules:
+  - name: blueprint-compiler
+    builddir: true
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/jwestman/blueprint-compiler.git
+        branch: main
```

Comment:
you need an explicit revision, not a branch

Reply:
Done now.